functions/controllers/expressExample/index.js

```diff
@@ -25,8 +31,35 @@ app.get("/", async (req, res) => {
   }
 });
 
+app.get("/resend", jwtCheck, async (req, res) => {
+  try {
+    res.json(req.user.sub);
+  } catch (error) {
+    res.status(500).send({ code: 500, message: "Bad/No key" });
+  }
+});
+
 app.post("/resend", jwtCheck, async (req, res) => {
-  res.status(201).send({ message: "private body reached" });
+  // this endpoint takes in a user_id then converts
+  // it into an API call
+  try {
+    const options = {
+      // options for the API call
+      method: "POST",
+      url: "https://cruzhacks.us.auth0.com/api/v2/jobs/verification-email",
+      headers: { authorization: "Bearer " + API_TOKENS.email },
+      data: {
+        user_id: req.user && req.user.id ? req.user.sub : "",
+      },
+    };
+
+    // make API call then response
+    res.status(201);
+    await axios(options);
+    res.send("Verification Email Sent");
```

Comment:
For client response, it might be better to send the response as one. Also, I see you added express.json(). It's still important to make sure we indicate the fields of our response so something like 
`res.status(201).send({message: "some message"})` so clients have a better time interfacing.